### ISSUE TITLE
Failed to install using python3.12

### ISSUE THREAD

**Issue by Opal18** [strodgers/epomaker-controller#25]
[epomakercontroller0.0.4_install_error.txt](https://github.com/user-attachments/files/16282568/epomakercontroller0.0.4_install_error.txt)
Attached log above.

I'm not a python user at all but is there any reason why this doesn't work using python3.12? I installed using 3.10 and everything is working fine now though lol

**Comment by strodgers** [comment]
hmm I tested with 3.12 and it does install

might be that you need a C++ compiler eg

```console
sudo dnf groupinstall "Development Tools"
sudo dnf install meson ninja-build
```

I'' test in a container and get back

**Comment by strodgers** [comment]
ok nvm I was being lazy with requirements and ended up with loads of unnecessary stuff in there
will release a new one with only the actual requirements

**Comment by strodgers** [comment]
I hope this is fixed now in  version 0.0.5, you can try with
```console
pip install EpomakerController==0.0.5
```

**Comment by strodgers** [comment]
assuming this is fixed
